packages/web-components/fast-foundation/src/patterns/start-end.ts

```diff
@@ -63,6 +63,7 @@ export const endSlotTemplate: (
         part="end"
         ${ref("endContainer")}
         class=${x => (definition.end ? "end" : void 0)}
+        @click="${(x, c) => x.handleClick(c.event as Event)}"
```

Comment:
I think we need to add the event handler imperativly via the `endContainer` ref. Adding the click handler to the template like this adds handle click behavior to *all* components that implement start / end and also assumes they have a method called 'handleClick', so this could easily throw at runtime.

Reply:
Got it! I'll make the change

Reply:
Ok I ended up adding the handler imperatively for all spans under the button control, let me know if it looks ok now @nicholasrice